src/generate-axios-client.test.ts

```diff
@@ -70,79 +69,7 @@ describe('generate', () => {
         },
       },
       expected: {
-        javascript: `/* eslint-disable */
-
-const substituteParams = (url, params) =>
-  Object.entries(params).reduce(
-    (url, [name, value]) => url.replace(":" + name, encodeURIComponent(value)),
-    url
-  );
-
-const removePathParams = (url, params) =>
-  Object.entries(params)
-    .filter(([key, value]) => value !== undefined)
-    .reduce(
-      (accum, [name, value]) =>
-        url.includes(":" + name) ? accum : { ...accum, [name]: value },
-      {}
-    );
-
-const parseQueryParamsFromPagingLink = (link) => {
-  const params = new URLSearchParams(link.split("?")[1]);
-
-  return {
-    nextPageToken: params.get("nextPageToken"),
-    pageSize: params.get("pageSize"),
-  };
-};
-
-class Client {
-  constructor(client) {
-    this.client = client;
-  }
-
-  getPosts(data, config) {
-    return this.client.request({
-      ...config,
-      method: "GET",
-      params: removePathParams("/posts", data),
-      url: substituteParams("/posts", data),
-    });
-  }
-
-  putPost(data, config) {
-    return this.client.request({
-      ...config,
-      method: "PUT",
-      data: removePathParams("/posts/:id", data),
-      url: substituteParams("/posts/:id", data),
-    });
-  }
-
-  async paginate(request, data, config) {
-    const result = [];
-
-    let nextPageParams = {};
-    do {
-      const response = await this[request.name](
-        { ...nextPageParams, ...data },
-        config
-      );
-
-      result.push(...response.data.items);
-
-      nextPageParams = response.data.links.next
-        ? parseQueryParamsFromPagingLink(response.data.links.next)
-        : {};
-    } while (!!nextPageParams.nextPageToken);
-
-    return result;
-  }
-}
-
-module.exports.Client = Client;
-`,
-        declaration: `/* eslint-disable */
+        typescript: `/* eslint-disable */
```

Comment:
do we still need this?

Reply:
I think we do since eslint does also check typescript files.